### DISCUSSION TITLE
How to change the table thresholds?

### DISCUSSION THREAD

**Discussion by kdubba** [deepdoctection/deepdoctection#225] [Q&A]
Hi all,
Since deepdoctection uses Table Transformer, what config setting corresponds to the "threshold" in the following HuggingFace code? Here, when I reduce the threshold, I get more tables that have low confidence.

image = Image.open(file_path).convert("RGB")
image_processor = AutoImageProcessor.from_pretrained("microsoft/table-transformer-detection")
...
results = image_processor.post_process_object_detection(outputs, **threshold=0.2**, target_sizes=target_sizes)[0]

Thanks,
Krishna

**Comment by JaMe76** [comment]
Hi,

changing the threshold by changing the config files is not possible, as the threshold is currently hard coded unfortunately. This parameter should belong in the model configs…

When invoking the  analyzer and suppose the table transformer model is the predictor of the first layout component you can get access to the parameter 

analyzer.pipeline_component_list[0].predictor.config.threshold
